ui/main.qml

```diff
@@ -62,7 +57,8 @@ ApplicationWindow {
     }
 
     Settings {
-        id: settings
+        id: appSettings
+        fileName: "data/qt/unknownUser"
```

Comment:
Instead of setting this via a signal in `appMain`, could we get this from a `QtProperty` in nim?